bitwarden_license/src/Sso/Utilities/Saml2BitHandler.cs

```diff
@@ -0,0 +1,199 @@
+using System.Text;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.Options;
+using Sustainsys.Saml2.AspNetCore2;
+using Sustainsys.Saml2.WebSso;
+
+namespace Bit.Sso.Utilities;
+
+public class Saml2BitHandler : IAuthenticationRequestHandler
```

Comment:
🎨 Maybe add a comment about where all this content came from -- I assume it's 99% copied.

Reply:
Will do, just wanted to validate this before cleaning it up 👍

Reply:
Comment added